### ISSUE TITLE
Project maintenance

### ISSUE THREAD

**Issue by MichalPB1** [hlohrenz/CI4-Relationship-Query-Builder#3]
@hlohrenz Good job! Is this project being / will be maintained?

**Comment by ismailceylan** [comment]
I have same concern too.

**Comment by MGatner** [comment]
The author appears to have no GitHub activity in the last year (https://github.com/hlohrenz).

While not a full ORM, we at [Tatter\Relations](https://github.com/tattersoftware/codeigniter4-relations) would be glad to have anyone looking for CodeIgniter 4 database relationships.

**Comment by eafarooqi** [comment]
Hi, @hlohrenz do you have any plans to continue this project ? no activity since July 2019

**Comment by mrelliot69** [comment]
Hi, @hlohrenz  .. is this still on the works ?

**Comment by hlohrenz** [comment]
Sorry as I do not use CodeIgniter for any of my personal projects, I fell out of touch with it. I still prefer Laravel so I have chosen to go the route I was comfortable with to begin with so leaves me little time to work on this without the passion being there.

Apologies! Somebody is free to take the code and start a new repo to revive what this package was trying to achieve.

**Comment by MGatner** [comment]
If you are interested in the project but don't have time to work on it I would recommend asking for maintainers and retaining this repo as the host - otherwise use tends to fraction wildly. 

If you are done with this and want to pass it on entirely then I would recommend finding a new owner and transferring the repo so Packagist and people's local repos follow.

**Comment by hlohrenz** [comment]
> If you are interested in the project but don't have time to work on it I would recommend asking for maintainers and retaining this repo as the host - otherwise use tends to fraction wildly.
> 
> If you are done with this and want to pass it on entirely then I would recommend finding a new owner and transferring the repo so Packagist and people's local repos follow.

Yup if somebody wants to take over ownership, then I'll gladly give it up. Haven't gotten any interests yet.
